hollow-ui-tools/src/main/java/com/netflix/hollow/ui/HollowUIRouter.java

```diff
@@ -32,8 +34,18 @@ public abstract class HollowUIRouter extends HttpServlet {
     protected final VelocityEngine velocityEngine;
 
     @Override
-    public abstract void doGet(HttpServletRequest request,
-        HttpServletResponse response) throws IOException;
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        try {
+            handle(request.getPathInfo(), request, response);
```

Comment:
The return type boolean isn't being checked. Why was the method signature changed?

Reply:
The commit message mentions the reason for refactoring. Did you see any issue which led to this refactoring?

Reply:
Did not discover any issues due to this, and did not see any usage of the return value within our code base, but retained the return value for backwards compatibility with versions upto 7.1